### PR TITLE
New comment endpoint works 

### PR DESCRIPTION
**Pull request by vlomeli**

### CODE REVIEW

File: src/controllers/posts.js [hunk 1]
```diff
@@ -21,3 +21,26 @@ export const createPost = async (req, res) => {
     });
   }
 };
+
+export const createComment = async (req, res) => {
+  const { comment_content } = req.body;
+
+  const { id: post_id } = req.params;
```
Comment (JJeshua) [inline src/controllers/posts.js:29]: Need to check whether the post id is valid (check if the post exists) and not null, otherwise server crashes

**Review by A13xYanez — APPROVED** [review]
Code has been tested and is functional. Creates the comment when the post id exists and returns status 404 when it does not.

**Comment by JJeshua** [comment]
Endpoint works as intended.

**Review by JJeshua — APPROVED** [review]
LGTM

**Pull request merged**